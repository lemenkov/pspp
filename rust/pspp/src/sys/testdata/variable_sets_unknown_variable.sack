# File header.
"$FL2"; s60 "$(#) SPSS DATA FILE PSPP synthetic test file";
2; # Layout code
10; # Nominal case size
0; # Not compressed
0; # Not weighted
0; # No cases.
100.0; # Bias.
"01 Jan 11"; "20:53:52"; s64 "PSPP synthetic test file";
i8 0 *3;

# Variable Set 1
2; 0; 0; 0; 0x050800 *2; i8 0x82; i8 0xa0; s6 "";
2; 0; 0; 0; 0x050800 *2; s8 "B";
2; 0; 0; 0; 0x050800 *2; s8 "C";

# vs2
2; 0; 0; 0; 0x050800 *2; s8 "D";
2; 0; 0; 0; 0x050800 *2; s8 "E";
2; 0; 0; 0; 0x050800 *2; s8 "F";
2; 0; 0; 0; 0x050800 *2; s8 "G";

# c
2; 4; 0; 0; 0x010400 *2; s8 "H";
2; 4; 0; 0; 0x010400 *2; s8 "I";
2; 4; 0; 0; 0x010400 *2; s8 "J";

# Machine integer info record.
7; 3; 4; 8; 1; 2; 3; -1; 1; 1; ENDIAN; 932;

7; 5; 1;
COUNT(
  "Variable Set 1= "; i8 0x82; i8 0xa0; " "; >>"xyzzy"<<; " b c"; i8 10;
  "vs2=d "; >>"foo"<<; " e f g"; i8 10;);

# Character encoding record.
7; 20; 1; 9; "shift_jis";

# Dictionary termination record.
999; 0;
