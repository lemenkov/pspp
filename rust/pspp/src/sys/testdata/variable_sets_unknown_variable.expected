Variable set "Variable Set 1" includes unknown variable xyzzy.

Variable set "vs2" includes unknown variable foo.

╭──────────────────────┬────────────────────────╮
│       Created        │    01-JAN-2011 20:53:52│
├──────────────────────┼────────────────────────┤
│Writer Product        │PSPP synthetic test file│
│       Version        │1.2.3                   │
├──────────────────────┼────────────────────────┤
│       Compression    │None                    │
│       Number of Cases│                       0│
╰──────────────────────┴────────────────────────╯

╭─────────┬────────────────────────╮
│Label    │PSPP synthetic test file│
│Variables│                      10│
╰─────────┴────────────────────────╯

╭──┬────────┬─────┬─────────────────┬─────┬─────┬─────────┬────────────┬────────────┬──────────────╮
│  │Position│Label│Measurement Level│ Role│Width│Alignment│Print Format│Write Format│Missing Values│
├──┼────────┼─────┼─────────────────┼─────┼─────┼─────────┼────────────┼────────────┼──────────────┤
│あ│       1│     │                 │Input│    8│Right    │F8.0        │F8.0        │              │
│b │       2│     │                 │Input│    8│Right    │F8.0        │F8.0        │              │
│c │       3│     │                 │Input│    8│Right    │F8.0        │F8.0        │              │
│d │       4│     │                 │Input│    8│Right    │F8.0        │F8.0        │              │
│e │       5│     │                 │Input│    8│Right    │F8.0        │F8.0        │              │
│f │       6│     │                 │Input│    8│Right    │F8.0        │F8.0        │              │
│g │       7│     │                 │Input│    8│Right    │F8.0        │F8.0        │              │
│h │       8│     │Nominal          │Input│    4│Left     │A4          │A4          │              │
│i │       9│     │Nominal          │Input│    4│Left     │A4          │A4          │              │
│j │      10│     │Nominal          │Input│    4│Left     │A4          │A4          │              │
╰──┴────────┴─────┴─────────────────┴─────┴─────┴─────────┴────────────┴────────────┴──────────────╯

╭───────────────────────────┬────────╮
│Variable Set and Position  │Variable│
├───────────────────────────┼────────┤
│Variable Set 1            1│あ      │
│                          2│b       │
│                          3│c       │
├───────────────────────────┼────────┤
│vs2                       1│d       │
│                          2│e       │
│                          3│f       │
│                          4│g       │
╰───────────────────────────┴────────╯
